.eslintrc.json

```
@@ -0,0 +1,17 @@
+{
+    "env": {
+        "browser": true,
```

This is `node` code, not browser code:
```suggestion
        "node": true,
```